pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-headers.adoc

```diff
@@ -0,0 +1,125 @@
+= Reading and Writing to Request Headers and Body
```

Comment:
```suggestion
= Reading and Writing Request Headers and Bodies
```
I'm thinking we don't read **to** request headers; rather we read **from** request headers? We probably don't need the "to".

Also "bodies" for parallelization.